src/components/settings/apiAccessTokens/CreateUpdateApiAccessToken.tsx

```diff
@@ -0,0 +1,343 @@
+import {
+  createApiAccessTokenValidationSchema,
+  updateApiAccessTokenValidationSchema,
+} from '@esss-swap/duo-validation/lib/Admin';
+import Button from '@material-ui/core/Button';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Grid from '@material-ui/core/Grid';
+import IconButton from '@material-ui/core/IconButton';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Tooltip from '@material-ui/core/Tooltip';
+import Typography from '@material-ui/core/Typography';
+import FileCopyIcon from '@material-ui/icons/FileCopy';
+import {
+  ErrorMessage,
+  Field,
+  FieldArray,
+  FieldArrayRenderProps,
+  Form,
+  Formik,
+} from 'formik';
+import { TextField } from 'formik-material-ui';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import SimpleTabs from 'components/common/TabPanel';
+import UOLoader from 'components/common/UOLoader';
+import { PermissionsWithAccessToken } from 'generated/sdk';
+import { useQueriesAndMutationsData } from 'hooks/admin/useQueriesAndMutationsData';
+import useDataApiWithFeedback from 'utils/useDataApiWithFeedback';
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    minHeight: '350px',
+    marginTop: '10px',
+    maxHeight: '550px',
+    overflowY: 'auto',
+    overflowX: 'hidden',
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+  error: {
+    color: theme.palette.error.main,
+    marginRight: '10px',
```

Comment:
You using `theme.spacing(...)` in other places, I think this and everything related to spacing should use it (margin*, padding*).